### ISSUE TITLE
UDSnackbar, add -Location, -fullsize and button

### ISSUE THREAD

**Issue by rwidmark** [ironmansoftware/powershell-universal#3535]
### Summary of the new feature / enhancement

It would be great if it was possible to add -Location to Snackbar.
Value should be left, right, top and bottom. (LeftButtom for example)
Also the possibility to have full size (screen wide) and a dismiss button would be perfect :)

Adding -Timeout where value is in ms for when it should disappear, and if you don't populate -Timeout it should be sticky until user click on dismiss button.

https://mui.com/material-ui/react-snackbar/

### Proposed technical implementation details (optional)

https://mui.com/material-ui/react-snackbar/

**Comment by adamdriscoll** [comment]
Show-UDSnackbar exists but it doesn't currently have positioning.

**Comment by rwidmark** [comment]
@adamdriscoll was looking under components at the webpage but could not find it.
Guess I did miss looking in the APIs :)

**Comment by rwidmark** [comment]
@adamdriscoll  Show-UDSnackbar are not in the cmdlets dock either. Just so you know :)

**Comment by adamdriscoll** [comment]
Yeah idk how this never made it into the docs. Will make sure we get that added too.

**Comment by rwidmark** [comment]
@adamdriscoll  added a wish for full-size and dismiss button / link also.

**Comment by adamdriscoll** [comment]
I've added everything but full size as you could use a modal for this and the snack bar doesn't support it natively. I did add  -ClassName so you could just change the size of it using CSS if you really want to do so.
